scripts/measures/get-inverse-measures-from-pdfs.js

```diff
@@ -0,0 +1,59 @@
+// running this will print out whether each measure is an inverse measure
+// claims pdfs can be downloaded from https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip
+// unzip and give the path to the Claims-Quality-Measures
+// usage ./get-inverse-measures-from-pdfs.js <path to folder of claims quality measures pdfs>
+
+const fs = require('fs');
+const program = require('commander');
+const pdfToText = require('pdf-to-text');
+const p = require('path');
+
+let folderPath = null;
+
+function setPath(path) {
+  folderPath = path;
+}
+
+program
+  .version('0.0.1')
+  .arguments('<path>')
+  .action(setPath)
+  .parse(process.argv);
+
+if (!folderPath) {
+  console.log('Missing required argument <path>');
+  process.exit(1);
+}
+
+function getIsInverse(path, cb) {
+  pdfToText.pdfToText(path, function(err, data) {
+    if (err) {
+      cb(false);
+      return;
+    }
+
+    let found = data.match(/inverse measure/i); // ignore case
+    let isInverse = false;
+
+    if (found) {
+      isInverse = true;
+    }
+
+    cb(isInverse);
+  });
+}
+
+// find all files in path
+// group the files by the qualityId. This way we don't have to look through the claims and registry files
+const groupedFiles = fs.readdirSync(folderPath).reduce(function(arr, current) {
+  let qualityId = current.split('_')[2];
+  arr[qualityId] ? arr[qualityId].push(current) : arr[qualityId] = [current];
+  return arr;
+}, {});
+
+const qualityIds = Object.keys(groupedFiles);
+qualityIds.forEach(function(qualityId, i) {
+  getIsInverse(p.join(folderPath, groupedFiles[qualityId][0]), function(isInverse) {
+    console.log(qualityId + ' : ' + isInverse);
```

Comment:
Is this writing into the measures data file directly? would it be possible to update it to automatically update to make it easier to keep it up to date?

Reply:
I think that directly manipulating the measure definition file goes against the approach taken in the rest of the repo. There are updates to the file that can be applied in `scripts/measures/enrich-measures-data.js` but it didn't really feel right to add behaviour here as it would be dependent on resources from outside the repository (the PDFs).

Reply:
@marimiyachi or @jiahuang will have a better final answer, but manually updating the file to include these inverse measure results doesn't seem like a sustainable solution to me.

Reply:
I understand now I think - see my other comment above 👍

Reply:
Yeah - this repo is in a really weird in between state right now as we are half way through a refactor. Preferred approach would be to add this as a function in `enrich-measures-data` and programmatically updated the fields. One approach that we've used in the past is the download/gitignore the zip (see [here](https://github.com/CMSgov/qpp-measures-data/blob/68742bc66ac383f5d155a682dc8426202d35ba91/scripts/measures/build-measures)) - that's probably the most straightforward approach - you could also write the results of this into a "staging" file - that might be more work than it's worth.